presto-docs/src/main/sphinx/release/release-0.242.rst

```diff
@@ -0,0 +1,60 @@
+=============
+Release 0.242
+=============
+
+**Highlights**
+==============
+* Add configuration property ``experimental.spiller.task-spilling-strategy`` for choosing different spilling strategy to use.
+* Add support for fragment result caching.
+* Add support for partition versioning.
+* Add support for caching HiveMetastore calls at a more granular level.
+
+General Changes
+_______________
+* Fix compiler error in ``LambdaBytecodeGenerator`` when common sub-expression optimization is enabled and multiple SQL functions contain lambda expressions.
+* Fix possible permanently stucked queries when memory limit is exceeded during shuffle.
+* Fix a bug where ``IS DISTINCT FROM NULL`` produces wrong results for columns with null values.
+* Add configuration property ``experimental.spiller.task-spilling-strategy`` for choosing different spilling strategy to use.
+* Add session property ``query_max_output_bytes`` and configuration property ``query.max-output-bytes`` to control how much data a query can output.
+* Add support for fragment result caching. This can be enabled by ``fragment-result-cache.enabled`` and ``experimental.fragment-result-caching-enabled``
+
+Hive Changes
+____________
+* Fix a bug where DWRF encryption would fail for large uncompressed column values.
+* Fix a bug where non-Presto readers could not read encrypted DWRF files written by Presto if the encryption group listed columns out of order.
+* Fix a performance regression for String field handling in GenericHiveRecordCursor when the SerDe does not provide an efficient Writable implementation.
+* Improve split loading efficiency by only using as many threads as are required.
+* Add support for file renaming for Hive connector. This can be enabled with the ``hive.file_renaming_enabled`` configuration property.
```

Comment:
Not a feedback to the release notes but I thought we normally use "-" rather than "_" in configs?

Reply:
@NikhilCollooru